script/zmb

```diff
@@ -613,6 +613,58 @@ sub cmd_get_batch_job_result {
 }
 
 
+=head2 batch_status
+
+ zmb [GLOBAL OPTIONS] batch_status [OPTIONS]
+
+ Options:
+   --batch-id BATCH-ID|--bi BATCH-ID
+   --list-waiting-tests|--lw
+   --list-running-tests|--lr
+   --list-finished-tests|--lf
```

Comment:
Maybe add negative options here as well to allow the sending of `false` arguments as well?